packages/@aws-cdk/aws-appsync/lib/graphqlapi.ts

```diff
@@ -248,6 +249,13 @@ export interface LogConfig {
    * @default - None
    */
   readonly role?: IRole;
+
+  /**
+  * log retention period
+  *
+  * @default - Use AppSync default
```

Comment:
Let's specify here what that default is, not that it's just the AppSync default.

Reply:
I hope it is better now :)